cmd/airflow.go

```diff
@@ -403,7 +406,9 @@ func newObjectExportCmd() *cobra.Command {
 	cmd.Flags().BoolVarP(&pools, "pools", "p", false, "Export pools to a settings file. Note pools cannot be exported to a env file ")
 	cmd.Flags().StringVarP(&settingsFile, "settings-file", "s", "airflow_settings.yaml", "The location of the file to store exported Airflow objects as YAML. Default is 'airflow_settings.yaml'")
 	cmd.Flags().BoolVarP(&envExport, "env-export", "n", false, "Export Airflow objects as Astro environment variables.")
+	cmd.Flags().BoolVarP(&compose, "compose", "", false, "Export compose file used in 'astro dev start'")
 	cmd.Flags().StringVarP(&envFile, "env", "e", ".env", "The location of the file to store exported Airflow objects as Astro environment variables. Default is '.env'.")
+	cmd.Flags().StringVarP(&composeFile, "compose-file", "", "compose.yaml", "The location to export the compose file used in 'astro dev start' to")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&exportComposeFile, "compose-file", "", "compose.yaml", "The location to export the compose file used in 'astro dev start' to")
```

Based on my above comment